### PR TITLE
chore: add back textarea

### PR DESCRIPTION
**Pull request by louis-bompart**
let's figure out why it's going haywire later on, eh.

### CODE REVIEW

**Review by fbeaudoincoveo — APPROVED** [review]
🤷

File: packages/atomic/src/pages/examples/commerce-website/search.html [hunk 1]
```diff
@@ -29,7 +29,7 @@ <h1>Search page</h1>
         <atomic-commerce-interface type="search" style="max-width: 1000px; margin: auto">
           <atomic-search-layout>
             <atomic-layout-section section="search">
-              <atomic-commerce-search-box>
+              <atomic-commerce-search-box textarea>
```
Comment (olamothe) [inline packages/atomic/src/pages/examples/commerce-website/search.html:32]: Not necessary, the search box should already be a textarea

Comment (fpbrault) [reply packages/atomic/src/pages/examples/commerce-website/search.html:32]: I think the problem is that we are still using the css of `atomic-search-box`

```css
@import '../../search/atomic-search-box/atomic-search-box.pcss';
```

if we remove the textarea attribute from the selector we should be good. Alternatively we could add textarea on the host attribute in `atomic-commerce-search-box` but I don't think thats the right solution.
```css
:host([textarea]) {
  @apply relative z-10;
}
```

Comment (louis-bompart) [reply packages/atomic/src/pages/examples/commerce-website/search.html:32]: Partially, other CSS rules apply only to textarea.

Comment (olamothe) [reply packages/atomic/src/pages/examples/commerce-website/search.html:32]: https://github.com/coveo/ui-kit/pull/3936/files#diff-91eb577ee22326dd19a78cfc762da30b9eb5b8819bbdaac2fec2adc49bc55b1bR2-R5 

+ the whole PR that adds sections

**Pull request closed without merging**